examples/core/opt-prop.py

```diff
@@ -74,7 +74,10 @@ def create_face(body):
 # is running as a service. In this example, the server and
 # client are the same machine.
 
-speos = Speos(host=HOSTNAME, port=GRPC_PORT)
+if USE_DOCKER:
+    speos = Speos(host=HOSTNAME, port=GRPC_PORT)
+else:
+    speos = launcher.launch_local_speos_rpc_server(port=GRPC_PORT)
```

Comment:
it should be "Start" a local RPC server in the description. Also, could you add description that the RPC server will not close in the end of the script unless using speos.close()

Reply:
i think i would rather add a speos close at the end